graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -243,6 +247,7 @@ public LookupTablePage tables(@ApiParam(name = "page") @QueryParam("page") @Defa
     public LookupTablePage get(@ApiParam(name = "idOrName") @PathParam("idOrName") @NotEmpty String idOrName,
                                @ApiParam(name = "resolve") @QueryParam("resolve") @DefaultValue("false") boolean resolveObjects) {
 
+        checkPermission(RestPermissions.LOOKUP_TABLES_READ, idOrName);
```

Comment:
This (and the other references to `idOrName`) can become problematic, if we ever grant individual permissions, because then it is not clear which one to use when editing the permission: the id or the name.

I think it is better to load the DTO before and then check the permission on the id and never the names.